scripts/reindex.py

```diff
@@ -199,6 +211,12 @@ def conda_data_wrapper(args):
         # write summary map of pypi package name to conda channel/name
         (PUBLIC / "conda.json").write_text(json.dumps(CONDA_INDEX, indent=2))
 
+        # update the main index (summary) with the conda versions basic sorting
+        # should match the method used in scripts/bigquery.py for PyPI
+        # versions. If you need fancy sorting downstream use 'pacakging'.
+        for pkg in PYPI_INDEX:
+            pkg["conda_versions"] = sorted(data.get(pkg["name"], {}).get("versions", []))
```

Comment:
I think a normal sort will not correctly sort versions like 0.1.1 vs 0.10.1

Reply:
I think you're right and I'd be in favor of adding that small dependency here for it. I'll make the change in the PR.

Reply:
Should be updated now - I think packaging was already being installed as a secondary dependency so now it's just explicit in `requirements.txt`.